src/main/java/it/gov/pagopa/pu/bff/service/org_sil_service/OrgSilServiceRetrieverServiceImpl.java

```diff
@@ -62,10 +65,45 @@ public PagedOrgSilServiceView getOrgSilServicesByFilters(Long organizationId, St
     return orgSilServiceViewMapper.map(
       orgSilServiceService.getOrgSilServicesByFilters(organizationId, applicationName, serviceType, flagLegacy, pageable, accessToken));
   }
+
   @Override
   public OrgSilServiceDecryptedDTO getOrgSilServiceDetails(Long organizationId, Long orgSilServiceId, UserInfo loggedUser, String accessToken) {
     authorizationService.validateAdminRole(organizationId, loggedUser);
 
     return orgSilServiceDTOMapper.map(orgSilServiceService.getOrgSilServiceByIdDecrypted(orgSilServiceId, accessToken));
   }
+
+  @Override
+  public OrgSilServiceDecryptedDTO createOrgSilService(Long organizationId, OrgSilServiceDecryptedDTO body, UserInfo loggedUser, String accessToken) {
```

Comment:
why a service called `Retriever` should perform create operation? let's create a new service instead

Reply:
thus you are using bad names for service :D
lets introduce a OrgSilServiceStorerService instead.